ext/bg/js/search-query-parser.js

```diff
@@ -59,7 +59,7 @@ class QueryParser extends TextScanner {
 
         this.search.setContent('terms', {definitions, context: {
             focus: false,
-            disableHistory: cause === 'mouse' ? true : false,
+            disableHistory: cause === 'mouse',
```

Comment:
I've seen `(...)` used for some places similar to this, should it be enforced too (if posssible with ESLint)?

Reply:
There is [`no-extra-parens`](https://eslint.org/docs/rules/no-extra-parens), which appears to do the opposite.

The reason why I sometimes do that is because, IMO, it more clearly delimits expressions, particularly multi-part ones (e.g. `(a === 3 && b === 4)`). It also separates places where there are similar operators (e.g. `let x = (y === 32)`).

However, I'm often not consistent about this, and I don't really have a preference about using either since the benefits are _very_ minor. I do think `no-extra-parens` would probably be overkill though; minor stylistic differences are fine so long as they don't harm readability or understandability. For example, I have seen lint rules which prefer to not have parentheses anywhere, _including_ around logic expressions with mixed `&&` and `||`. Example: `x && y || z` is less clear than `(x && y) || z`.

Reply:
Yeah, it can be very contextual when it makes sense to add the extra parentheses, not something that a linter rule will understand. There are also code formatters like [Black](https://github.com/psf/black) (Python) that automate the part of fixing lint errors, but having tried that there are often cases when an alternative way of formatting would make it more clear for future readers. The benefit is that you don't have to spend time thinking of formatting.